monitor/grafana/zeebe.json

```diff
@@ -3391,7 +3391,7 @@
             "h": 8,
             "w": 12,
             "x": 0,
-            "y": 4
+            "y": 5
```

Comment:
did these `y` value have to change as well? or does it not really matter?

Reply:
:shrug: it is generated idk :laughing:

Reply:
well... perhaps you moved something around before the change? It reminds me of making a change to a bpmn file using the eclipse plugin on 2 different computers. Will always result in changes in the locations of elements. 😅

Reply:
actually this happens always if you export the file I have the feeling :sweat_smile: